pom.xml

```
@@ -251,8 +286,7 @@
 
 	<properties>
 		<findbugs.failOnError>false</findbugs.failOnError>
-				<workflow.version>1.6</workflow.version>
-                <jenkins.version>2.120</jenkins.version>
+                <jenkins.version>2.319.3</jenkins.version>
```

```suggestion
                <jenkins.version>2.319.1</jenkins.version>
```
There's been a lot of back and forth about point releases of LTS lines, but I'd prefer to use .1 for now.